documentation/docs/student/quiz-training-mode.mdx

```diff
@@ -0,0 +1,200 @@
+- **Correct Answers:** Total number of correct answers
+- **Wrong Answers:** Total number of wrong answers
+
+<Image src={scorecard} alt={"Scorecard"} size={ImageSize.small}></Image>
+
+### Leaderboard
+
+The right side shows the course leaderboard (if enabled):
+
+- **Your Position:** Highlighted with your rank, name, and score
+- **Top Participants:** Leaderboard of other course participants
+- **Profile Pictures:** Visual representation of participants
+
+<Image src={leaderboard} alt={"Leaderboard"} size={ImageSize.large}></Image>
+
+---
+
+## Starting Training
+
+Click **"Start Training"** to begin practicing.
+
+### Answering Questions
+
+The system presents questions in three different formats:
+
+#### Multiple-Choice Questions
+
+- Select one or more correct answers
+- Submit your solution when finished
+
+<Image src={mcQuestion} alt={"MC Question"} size={ImageSize.large}></Image>
+
+#### Drag-and-Drop Questions
+
+- Drag elements to the correct position
+- Submit your solution when finished
+
+<Image src={ddQuestion} alt={"DD Question"} size={ImageSize.large}></Image>
+
+#### Short Answer Questions
+
+- Enter text answers in the provided fields
+- Pay attention to correct spelling
+- Submit your solution when finished
+
+<Image src={saQuestion} alt={"SA Question"} size={ImageSize.large}></Image>
+
+### Graded vs. Ungraded Questions
+
+<Image src={unratedNotification} alt={"Unrated Notification"} size={ImageSize.medium}></Image>
+
+<Callout variant={CalloutVariant.info}>
+        When an ungraded question appears, you'll be asked if you want to continue.
+        Ungraded questions don't count toward your score but help with practice.
+</Callout>
+
+### Receiving Feedback
+
+After submitting, you'll immediately see:
+
+- **Score Achieved:** How many points you earned for this question
+- **Correct Answers:** Which options were correct (marked in green)
+- **Wrong Answers:** Which options were incorrect (marked in red)
+- **Explanations:** Additional hints about the correct solution
+
+<Image src={mcQuestionSolution} alt={"Solution"} size={ImageSize.large}></Image>
+
+- **Sample Solution:** A button to view a sample solution (for short-answer and drag-and-drop questions)
+
+<Image src={saQuestionSolution} alt={"Sample Solution"} size={ImageSize.large}></Image>
+
+### Navigation
+
+- **Next Question:** Proceed to the next question
+- **End Training:** Return to the overview page (appears on the last question)
+
+---
+
+## Leaderboard Settings
+
+You can adjust your leaderboard visibility at any time:
+
+- **From the Overview Page:** Click the **"View description"** button in the scorecard to open the description dialog and settings
+- **From User Settings:** Navigate to your user settings and change the quiz training leaderboard visibility
+
+<Image src={userSettings} alt={"User Settings"} size={ImageSize.small}></Image>
+<Image src={userLeaderboardSettings} alt={"User Leaderboard Settings"} size={ImageSize.medium}></Image>
+
```

Comment:
I think this should be included when the leaderboard settings are mentioned for the first time